bagri-server/bagri-server-hazelcast/src/main/java/com/bagri/server/hazelcast/impl/DocumentManagementImpl.java

```diff
@@ -1319,6 +1304,10 @@ public int removeDocumentFromCollections(String uri, String[] collections) {
 		return remCount;
 	}
 
+	public void deleteDocumentFromContentCache(Map.Entry<DocumentKey, Document> entry) {

```

Comment:
add another method which will be used from processor and will do all logic, instead of this one

Reply:
done